doc/en/installation/common/sql_strict_mode.rst

```diff
@@ -0,0 +1,29 @@
+.. warning::
+    Centreon started the compatibility with SQL strict mode but not all components are ready yet. It is mandatory to
+    disable the strict mode if you use MariaDB >= 10.2.4 or MySQL >= 5.7.5.
+    
```

Comment:
"Notice :You can enable the strict mode for testing purpose, but you shouldn't do so on your production environment as it's still in a beta stage"